temba/contacts/models.py

```diff
@@ -2148,6 +2151,9 @@ def try_to_parse(cls, org: Org, file, filename: str) -> Tuple[List, int]:
                 seen_uuids.add(uuid)
             for urn in urns:
                 if urn in seen_urns:
```

Comment:
I think we can add this statement unique like:

```
if urn in seen_urns:
    if not settings.ALLOW_DUPLICATE_CONTACT_IMPORT:
        raise ValidationError(
            _("Import file contains duplicated contact URN '%(urn)s'."), params={"urn": urn}
        )

    num_duplicates += 1
```
What do you think?

Reply:
yup, looks legible that way, got my buried in the tests for a while now